src/main/java/core/basesyntax/strategy/handler/ReturnHandler.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax.strategy.handler;
+
+public class ReturnHandler extends IncrementHandler {
```

Comment:
Don't begin class implementation with an empty line. The empty line after the package declaration (line 2) should be removed to comply with the checklist.